### DISCUSSION TITLE
Selecting multiple nodes does not highlight the selection on the 2d scene

### DISCUSSION THREAD

**Discussion by PereViader** [godotengine/godot-proposals#7960] [2D]
Godot v4.1.1

Steps to reproduce
- Create a new empty 2d Scene
- Add several Sprite nodes with visuals to it
- Use the mouse to select an area that encapsulates more than one of those sprites

What happens:
- Those nodes are selected on the "hierarchy" tree
- Those nodes have no visuals representing they are selected on the 2d scene

What is expected:
- Those nodes are selected on the "hierarchy" tree
- Those nodes have some selection visuals representing they are selected on the 2d scene

Selecting a single node does show the "transform" gizmo.
When more than one node is selected they properly move together as expected, but have no visuals to easily see that the selection was done properly.

![image](https://github.com/godotengine/godot-proposals/assets/32639021/edcf07bd-28f0-40b4-9d1e-ef44722ad281)
![image](https://github.com/godotengine/godot-proposals/assets/32639021/a2f084cb-600e-4480-9c0f-0de80659e112)

**Comment by Calinou** [comment]
> When more than one node is selected they properly move together as expected, but have no visuals to easily see that the selection was done properly.

There is an orange outline around each selected item, but it's not very visible on your example due to the sprites having solid borders.

The resize gizmos are also only available when a single node is selected. It's likely possible to implement support for resizing multiple nodes at once using gizmos, but it's nontrivial.

**Reply by PereViader** [reply]
You are actually right, there is a an outline I didn't notice.
Closing. Thanks!
